src/components/MultiHopTrade/components/TradeInput/TradeInput.tsx

```diff
@@ -281,6 +281,7 @@ export const TradeInput = ({ isCompact, tradeInputRef, onChangeTab }: TradeInput
     },
     [dispatch],
   )
+  console.log({ buyAsset })
```

Comment:
ser

Reply:
Sry in [3d19513](https://github.com/shapeshift/web/pull/7992/commits/3d1951386bf5e0dba41a1d9082f2a0598502f0ae)